controllers/greptimedbstandalone/deployer.go

```diff
@@ -253,6 +253,10 @@ func (b *standaloneBuilder) BuildStatefulSet() deployer.Builder {
 			},
 			Template:             b.generatePodTemplateSpec(),
 			VolumeClaimTemplates: b.generatePVCs(),
+			UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
+				Type:          appsv1.RollingUpdateStatefulSetStrategyType,
```

Comment:
Can we support StatefulSet updateStrategy type of OnDelete?
https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies

Reply:
I don't think we need `OnDelete`. It's a legacy option. We always use rolling update.